Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/tools.md

---
title: "Ceres Tools | SORA Docs"
head:
  - - meta
    - name: description
      content: "Ceres Tools is a platform supported by the SORA community which displays data from the Polkaswap DEX."
  - - meta
    - name: keywords
      content: "Ceres, DeFi, tools, tokens, pairs, liquidity, prices, portfolio, swaps, charts, Polkaswap, SORA network"
---

# Ceres Tools

[Ceres Tools](https://tools.cerestoken.io) is a platform supported by the SORA community which displays data from the Polkaswap DEX. 

> Users are able to track tokens prices, pairs liquidity, pool details, wallets, portfolio, swaps, charts, volumes etc. 

Reviewer: ```suggestion
Users can track token prices, liquidity pairs and pool details, wallets, portfolios, swaps, charts, volumes, etc. 
```

> The platform is also available in the form of a mobile application for [Android](https://play.google.com/store/apps/details?id=io.cerestoken.tools) and [iOS](https://apps.apple.com/gb/app/ceres-tools/id1590032285).

Reviewer: ```suggestion
The platform is also available as a mobile application for the [Android](https://play.google.com/store/apps/details?id=io.cerestoken.tools) and [iOS](https://apps.apple.com/gb/app/ceres-tools/id1590032285) operating systems.
```

## Tokens

> The prices of tokens listed on Polkaswap, their asset IDs, token locks and token holders are shown in the Tokens section.

Reviewer: ```suggestion
Token prices listed on Polkaswap, their asset IDs, token locks, and token holders are displayed in the Tokens section.
```

## Pairs

> Pairs section shows liquidity, 24hr volume and pool details for each pair as

Reviewer: ```suggestion
The pairs section shows liquidity, 24 hour volume, and pool details for each trading pair as
```

> well as total liquidity, total 24hr volume on Polkaswap and details about liquidity locks.

Reviewer: ```suggestion
well as total liquidity, total 24-hour volume on Polkaswap, and details about liquidity locks.
```

> Additionally, in this section, users can check Deposit/Withdraw liquidity events and liquidity changes history for each pair.

Reviewer: ```suggestion
In this section, users can also check Deposit/Withdrawal liquidity events and liquidity change history for each pair.
```

## Farming

> Farming section shows details about all farmings and stakings on SORA in one place. Users can check Demeter, Hermes and PSWAP farmings/stakings.

Reviewer: ```suggestion
The farming section details all farming and staking on SORA in one place. Users can check Demeter, Hermes, and PSWAP farming/staking.
```

## Burning Tracker

In this section, users can track PSWAP and VAL burning.

> ## Charts&Swaps

Reviewer: ```suggestion
## Charts & Swaps
```

> In the Charts part of the section, users can track price movement of tokens

Reviewer: ```suggestion
In the Charts part of the section, users can track the token price movement
```

> listed on Polkaswap as well as doing a Technical Analysis using various

Reviewer: ```suggestion
listed on Polkaswap, as well as perform a Technical Analysis using various
```

> indicators. In the Swaps part of the section, users are able to track swaps

Reviewer: ```suggestion
indicators. In the Swaps part of the section, users can track swaps
```

(buys and sells) for each token on Ceres Tools.

## Portfolio

> A portfolio feature allows users to track wallet’s

Reviewer: ```suggestion
The portfolio feature allows users to track a wallets
```

> holdings, staking, rewards, liquidity, swaps and transfers. Also users can

Reviewer: ```suggestion
holdings, staking, rewards, liquidity, swaps, and transfers. Users can also
```

> monitor the movement of the portfolio in the previous 1h, 24h, 7d and

Reviewer: ```suggestion
monitor the portfolio's movement in 1h, 24-hour, 7-day and
```

> 30d.

Reviewer: ```suggestion
30-day intervals.
```

## TBC Reserves

> In this section, users can track TBC (Token Bonding Curve) reserves.

Reviewer: ```suggestion
Users can track the SORA TBC (Token Bonding Curve) reserves in this section.
```
